src/components/Button/Button.module.css

```diff
@@ -23,6 +25,11 @@
   cursor: pointer;
   transition: all var(--eds-anim-fade-quick) var(--eds-anim-ease);
 
+  /* TODO: make default color prop in Icon inherit or something? */
+  svg {
+    fill: currentColor;
+  }
```

Comment:
Not sure what we want to do here. I figure using the default `Icon` color prop would be best because then it's easy for the developer to override it if needed (of course this is dependent on the `color` prop being implemented in the `Icon` component).

Reply:
ask and ye shall receive 🙂 https://github.com/chanzuckerberg/edu-design-system/pull/940/files#diff-08acd9c461dd2b3858a54d0dd361b9164bdc650ee0050ace49c108d9dc6e2059